inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/debezium/DebeziumSourceFunction.java

```diff
@@ -408,12 +417,17 @@ public void run(SourceContext<T> sourceContext) throws Exception {
                 (Gauge<Long>) () -> debeziumChangeFetcher.getEmitDelay());
         metricGroup.gauge(
                 "sourceIdleTime", (Gauge<Long>) () -> debeziumChangeFetcher.getIdleTime());
-        if (StringUtils.isNotEmpty(this.inlongMetric)) {
-            String[] inlongMetricArray = inlongMetric.split(Constants.DELIMITER);
-            String groupId = inlongMetricArray[0];
-            String streamId = inlongMetricArray[1];
-            String nodeId = inlongMetricArray[2];
-            sourceMetricData = new SourceMetricData(groupId, streamId, nodeId, metricGroup);
+        if (StringUtils.isNotEmpty(this.inLongMetric)) {
+            String[] inLongMetricArray = inLongMetric.split(Constants.DELIMITER);
```

Comment:
Please keep the naming of `inlongMetric` and `inlongMetricArray`, and change the `inLongAudit` to `inlongAudit`, thanks.